src/StructuredLogger/BinaryLogger/BinaryLogger.cs

```diff
@@ -55,7 +55,7 @@ public sealed class BinaryLogger : ILogger
         //   - TargetSkippedEventArgs: added OriginallySucceeded, Condition, EvaluatedCondition
         // version 14:
         //   - TargetSkippedEventArgs: added SkipReason, OriginalBuildEventContext
-        internal const int FileFormatVersion = 14;
+        internal const int FileFormatVersion = 15;

```

Comment:
Don't think we need to rev the file format version here.

We need to ensure that we keep the file format version in sync between MSBuild and this repo. I guess the file format doesn't really change. Let's see if we find another way.

Reply:
Also this wouldn't really help anyway because the binlog files are recorded by MSBuild and not by this repo, so changing this repo's version would only impact binlogs recorded by this logger (which is 0.1% of cases maybe).